### PR TITLE
[master] Fix download url problem with miner and seed node join config

### PR DESCRIPTION
**Pull request by chetan-zilliqa**
## Backward Compatibility

- [ ] This is not a breaking change
- [ ] This is a breaking change

### Implementation

- [ ] **ready for review**

### Integration Test (Core Team)

- [ ] local machine test

- [ ] small-scale cloud test

### CODE REVIEW

**Review by mauromedda — APPROVED** [review]

File: scripts/download_incr_DB.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def getURL():
 	elif AWS_ENDPOINT_URL:
 		return f"{AWS_ENDPOINT_URL}/{BUCKET_NAME}"
 	else:
-		return "http://"+BUCKET_NAME+".s3.amazonaws.com"
+		return "http://"+BUCKET_NAME+".storage.googleapis.com"
```
Comment (yaron-zilliqa) [inline scripts/download_incr_DB.py:71]: Why? This seems incorrect. The whole point was to also preserve the AWS functionality.

**Review by yaron-zilliqa — CHANGES_REQUESTED** [review]

File: scripts/download_incr_DB.py (hunk 1 above)
Comment (rrw-zilliqa) [reply scripts/download_incr_DB.py:71]: @yaron-zilliqa  - if you'd checked the conversation in slack, you'd know why @chetan-zilliqa made this change. @chetan-zilliqa  - if you'd linked this PR to the JIRA issue, @yaron-zilliqa would've known why you did it, and if you'd linked to the slack discussion in the issue, he'd've been able to find the relevant conversation and probably come up with a fix.

Comment (mauromedda) [reply scripts/download_incr_DB.py:71]: For reference: https://zilliqa-jira.atlassian.net/jira/software/c/projects/ZIL/issues/ZIL-5453
The ticket to add the support on testnet repo for both CDN and GCP has been added into the backlog https://zilliqa-jira.atlassian.net/jira/software/c/projects/ZIL/issues/ZIL-5455

**Comment by chetan-zilliqa** [comment]
Closing the PR as the issue is fixed in another testnet merge. - https://github.com/Zilliqa/testnet/pull/1056

**Pull request closed without merging**